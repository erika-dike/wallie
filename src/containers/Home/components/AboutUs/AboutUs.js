import React from 'react';

import './AboutUs.css';


const AboutUs = () =>
  <div className="AboutUs">
    <div className="flex-module">
      <h3>
        <span className="AboutUs-title">About</span>
      </h3>
      <p>
        Wallie is a platform that allows users share short posts with the world.
        It accepts text, links, videos and images.
      </p>
    </div>
  </div>;

export default AboutUs;
