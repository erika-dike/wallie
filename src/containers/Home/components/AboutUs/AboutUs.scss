@import '../../../../shared.scss';

.AboutUs {
    background-color: #fff;
}

.AboutUs-title {
    @include section-title();
}

.flex-module {
    border-radius: 5px;
    color: #657786;
    font-size: 12px;
}
