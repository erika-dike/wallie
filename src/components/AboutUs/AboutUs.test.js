import React from 'react';
import renderer from 'react-test-renderer';

import AboutUs from './AboutUs';


it('renders correctly', () => {
  const tree = renderer.create(<AboutUs />).toJSON();
  expect(tree).toMatchSnapshot();
});
