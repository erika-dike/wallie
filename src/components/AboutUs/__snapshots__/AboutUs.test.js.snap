exports[`test renders correctly 1`] = `
<div
  className="AboutUs">
  <div
    className="flex-module">
    <h3>
      <span
        className="AboutUs-title">
        About
      </span>
    </h3>
    <p>
      Wallie is a platform that allows users share short posts with the world. It accepts text, links, and images.
    </p>
  </div>
</div>
`;
